### ISSUE TITLE
Giving priority to certain plugins

### ISSUE THREAD

**Issue by fadelm0** [Wox-launcher/Wox#1807]
Is there any way to give priority to certain plugins in searches?

What happens is when I try searching using the [bang search plugin](http://www.getwox.com/plugin/140), a whole bunch of results appear first from "Everything", and the bang search itself appears last, unless I type in some really obscure combination of characters.

I think that what should happen is as soon as I type in the ! character, it should automatically give priority to the bang search plugin. I have no idea if this issue happens because of Wox itself or because of the plugin.

**Comment by lances101** [comment]
Currently it entirely depends on the weight the plugin gives to the results. 
Your best option right now is to do go to the plugin's setting page and add an action keyword for that plugin.
For example, you could set it it to "!" and then the wpm query would become `!!yt cats`. 

I have not used the bang search plugin myself and therefore don't know if there is a configuration option to remove the bang expected by the plugin (although I suppose its not).

**Comment by fadelm0** [comment]
Although it sounds like it should work, unfortunately it didn't. Adding any action keyword stops the plugin from working. I suppose it has to do with the way it is programmed. I took a look at it, and tried changing a few things, but I don't have enough experience with either python or the Wox API to effectively change the way it works.

Is there a way I could contact the developer of this plugin? The plugins page didn't seem to provide a way, although maybe I just missed it.

**Comment by lances101** [comment]
Sadly, the developer did not leave a link to a github repository.
Anyway, I took a quick look and there seems to be a way to easily fix this. Go to the plugins directory at`%appdata%\Wox\Plugins\Bang Search` and replace the contents of the file [with this](
https://gist.githubusercontent.com/lances101/2f850ff04067cc0b7601d48f9e0dee1d/raw/7c8cc6e33a86071fca685ec488f72ff971bd8332/main.py).

I added the `Score` property to the result. This should ensure that Bang Search is the top result.

Should work. Reopen if you still have problems.
